app/models/form_profiles/va288832.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+class FormProfiles::VA288832 < FormProfile
+
+  class FormAddress
```

Comment:
```suggestion
class FormProfiles::VA288832 < FormProfile
  class FormAddress
```

Reply:
Since we are only making use of existing infrastructure I heavily changed this code